libcloud/test/compute/test_ecs.py

```diff
@@ -940,6 +955,20 @@ def _create_public_ip_AllocatePublicIpAddress(self, method, url, body, headers):
         resp_body = self.fixtures.load('create_public_ip.xml')
         return (httplib.OK, resp_body, {}, httplib.responses[httplib.OK])
 
+    def _ex_security_group_ingress_permission_AuthorizeSecurityGroup(
+            self, method, url, body, headers):
+        resp_body = self.fixtures.load('security_group_ingress.xml')
+        return (httplib.OK, resp_body, {}, httplib.responses[httplib.OK])
+
+    def _ex_security_group_egress_permission_AuthorizeSecurityGroupEgress(
+            self, method, url, body, headers):
+        resp_body = self.fixtures.load('security_group_egress.xml')
+        return (httplib.OK, resp_body, {}, httplib.responses[httplib.OK])
+
+
+if __name__ == '__main__':
```

Comment:
Looks like this some how accidentally ended up here and now it's duplicated twice :)

Should remove one set.